CryptoTechReminderSystem.AcceptanceTest/CryptoTechReminderSystemTests.cs

```diff
@@ -111,5 +117,74 @@ public void CanRemindLateDevelopersAtTenThirtyOnFriday()
                 _slackApi.ReceivedRequests[i].Url.Should().Be(SlackApiAddress + SlackApiPostMessagePath);
             }
         }
+        
+        
+        [TestFixture]
+        public class BetweenTenThirtyAndOneThirty
+        {
+            [OneTimeSetUp]
+            public void Init()
+            {
+                HandleSetUp();
+            }
+            
+            [Test]
+            [TestCase(10, 45, 0)]
+            [TestCase(11, 00, 4)]
+            [TestCase(11, 15, 4)]
+            [TestCase(11, 30, 8)]
+            [TestCase(11, 45, 8)]
+            [TestCase(12, 00, 12)]
+            [TestCase(12, 15, 12)]
+            [TestCase(12, 30, 16)]
+            [TestCase(12, 45, 16)]
+            [TestCase(13, 00, 20)]
+            [TestCase(13, 15, 20)]
+            public void CanRemindLateDevelopersEveryHalfHourUntilOneThirty(int hour, int minute, int expectedCount)
+            {                      
+                var getLateDevelopers = new GetLateDevelopers(_slackGateway, _harvestGateway, _harvestGateway);
+                var clock = new ClockStub(
+                    new DateTimeOffset(
+                        new DateTime(2019, 03, 01, hour, minute, 0)
+                    )
+                );
+
+                var remindLateDevelopers = new RemindLateDevelopers(getLateDevelopers, _remindDeveloper, clock);
+
+                remindLateDevelopers.Execute(new RemindLateDevelopersRequest
+                    {
+                        Message = "Please make sure your timesheet is submitted by 13:30 on Friday."
+                    }
+                );
+               
+                _slackApi.ReceivedRequests.Count.Should().Be(expectedCount);
+            }  
+        }
+
+        [Test]
+        public void CanRemindLearnTechChannelAtOneThirty()
+        {
+            var getLateDevelopers = new GetLateDevelopers(_slackGateway, _harvestGateway, _harvestGateway);
+            
+            var clock = new ClockStub(
+                new DateTimeOffset(
+                    new DateTime(2019, 03, 01, 13, 30, 0)
+                )
+            );
+
+            var shameLateDevelopers = new ShameLateDevelopers(getLateDevelopers, _remindDeveloper, clock);
+            
+            shameLateDevelopers.Execute(new ShameLateDevelopersRequest
+                {
+                    Message = ":bell: give me the sheets yo :bell:",
+                    Channel = "CHBUZLJT1"
+                }
+            );
+
+            JObject.Parse(_slackApi.ReceivedRequests.Last().RequestBody)["channel"]
+                .ToString().Should().Be("CHBUZLJT1");
+            JObject.Parse(_slackApi.ReceivedRequests.Last().RequestBody)["text"]
+                .ToString().Should().Be(":bell: give me the sheets yo :bell:\n• <@W123AROB>\n• <@W345ABAT>\n• <@W345ALFR>"); 
```

Comment:
This long line looks messy, it can be split into multiple lines.

Reply:
I tried to improve this in https://github.com/chubberlisk/crypto_tech/pull/17/commits/0902383871a67c20b5a700db3dbb51c6dea6699c. Any better?

Reply:
Yes, this is better. Thank you.